src/node.cc

```diff
@@ -2939,6 +2939,13 @@ void SetupProcessObject(Environment* env,
   READONLY_PROPERTY(versions,
                     "icu",
                     OneByteString(env->isolate(), U_ICU_VERSION));
+
+  if(icu_data_dir != nullptr) {
```

Comment:
I'm not 100% sure if cpplint complains about it but there should be a space after the `if`.